src/main/java/org/truffleruby/core/binding/BindingNodes.java

```diff
@@ -395,4 +401,20 @@ public DynamicObject allocate(DynamicObject rubyClass) {
 
     }
 
+    @Primitive(name = "caller_binding")
```

Comment:
I think the error in `jt lint` on Travis is due to this node missing `needsSelf=false`.
I'll check it and improve the error message.